contracts/staking/SimpleStaking.sol

```diff
@@ -247,10 +249,10 @@ contract SimpleStaking is DSMath, Pausable, AbstractGoodStaking, StakingToken{
 	{
 		uint256 er = exchangeRate();
 		uint256 tokenWorth = currentTokenWorth();
-		if (tokenWorth <= interestData.globalTotalStaked) {
+		if (tokenWorth <= totalProductivity) {
 			return (0, 0, 0);
 		}
-		uint256 tokenGains = tokenWorth.sub(interestData.globalTotalStaked);
+		uint256 tokenGains = tokenWorth.sub(totalProductivity);
 		(uint256 decimalDifference, bool caseType) = tokenDecimalPrecision();
 		//mul by `10^decimalDifference` to equalize precision otherwise since exchangerate is very big, dividing by it would result in 0.
 		uint256 iTokenGains;
```

Comment:
the if(casetype) below has exactly the same line for both cases

Reply:
I did not understand what you mean @sirpy

Reply:
my bad...